cli/azd/internal/cmd/cmd_help.go

```diff
@@ -0,0 +1,42 @@
+package cmd
```

Comment:
I duplicated the parts of `cmd_help.go` and `util.go` that we needed here.  I am happy to export these and then remove the copies from `cmd` (in favor of pointing at the ones here), I just didn't do that yet because I wanted to do this refactoring quickly and tear it off from my larger WIP run.

Happy to clean this up either now or later - let me know what the team wants me to do, @vhvb1989, @weikanglim.